apps/dot_ext/loggers.py

```diff
@@ -0,0 +1,203 @@
+import uuid
+from django.db.utils import IntegrityError
+from oauth2_provider.models import get_application_model
+from .models import AuthFlowUuid
+
+"""
+  Logger related functions for dot_ext/mymedicare_cb modules and auth flow trace logging.
+"""
+
+SESSION_AUTH_FLOW_TRACE_KEYS = ['auth_uuid', 'auth_client_id', 'auth_app_id', 'auth_app_name', 'auth_pkce_method']
+
+
+def cleanup_session_auth_flow_trace(request=None):
+    '''
+    Function for cleaning up auth flow related items
+    in session.
+    '''
+    if request:
+        # We are done using auth flow trace values, clear them from the session.
+        for k in SESSION_AUTH_FLOW_TRACE_KEYS:
+            try:
+                del request.session[k]
+            except KeyError:
+                pass
+
+
+def create_session_auth_flow_trace(request):
+    '''
+    Function for creating auth flow log tracing related items.
+
+    - Creates a new AuthFlowUuid instance.
+    - Sets new auth flow values in session.
+
+    Returns an AuthFlowUuid instance or None.
+    '''
+    if request:
+        auth_flow_uuid = None
```

Comment:
Should this variable be moved out of the if statement to the function will return properly even if request is None?

Reply:
@njdister Great catch! I removed that `auth_flow_uuid` return in the function, since it is not used.